one.ablaze.floorp.yml

```diff
@@ -0,0 +1,45 @@
+app-id: one.ablaze.floorp
```

Comment:
the ustrem tarball use `org.ablaze.floorp` to install icons and the desktop file. And that `.org` domain seem to not exist. so renaming need to be done. (and possibly changes to rename them submitted upstream along with the appdata file)

Reply:
OK
Updated upstream tarball and updated checksums.